### PR TITLE
Validation: Added Missing Descriptions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing metadata descriptions.

### CODE REVIEW

**Comment by dereklh7** [comment]
@stevemunk @JuanPapas 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by dereklh7** [comment]
@stevemunk @JuanPapas

I've resolved all merge conflicts for this PR. Please let me know if there are any other issues preventing approval and merge for this PR. Thanks!

**Comment by stevemunk** [comment]
@dereklh7 It looks like there are still conflicts.

**Comment by dereklh7** [comment]
@stevemunk I've resolved all merge conflicts. Please let me know if there are any remaining issues on this PR preventing approval and merge. Thank you and have a great day!

File: BingMaps/sdk-native/map-control-concepts/map-icons.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
-title: "Map Icons | Microsoft Docs"
-ms.author: "pablocan"
+title: Map Icons | Microsoft Docs
+description: Describes icons's utility, outlines how to display icons on the map, and provides additional references.
```
Comment (stevemunk) [inline BingMaps/sdk-native/map-control-concepts/map-icons.md:3]:
```suggestion
description: Describes Icons, the primary way to add a graphical image and text at a location within the map using the Bing Maps native controls, with a link to examples and additional information.

  utility, outlines how to display icons on the map, and provides additional references.
```

File: BingMaps/sdk-native/map-control-concepts/map-polylines-and-polygons.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
-title: "Map Polylines and Polygons | Microsoft Docs"
-ms.author: "pablocan"
+title: Map Polylines and Polygons | Microsoft Docs
+description: Describes polylines and polygons and provides additional resources and examples to provide guidance for drawing with polylines and polygons.
```
Comment (stevemunk) [inline BingMaps/sdk-native/map-control-concepts/map-polylines-and-polygons.md:3]:
```suggestion
description: Describes polylines and polygons with a link to examples and additional information.
```

File: BingMaps/sdk-native/map-control-concepts/map-styles-sheets.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Map Style Sheets"
-description: Learn how to use a Map Style Sheet to represent a set of rules that define the style of the map.
-ms.author: "pablocan"
+title: Style Sheets | Microsoft Docs
+description: Describes map style sheets and the MapStyleSheet static methods and provides information on default and custom map style sheets.
```
Comment (stevemunk) [inline BingMaps/sdk-native/map-control-concepts/map-styles-sheets.md:3]:
```suggestion
description: Describes map style sheets, which represent a set of rules that define the style of the map in a map view, including a table listing the default styles such as Aerial, RoadDark and RoadHighContrastDark and links to articles describing how to create custom map styles.

```

File: BingMaps/sdk-native/map-services-api/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "MapServices API Reference | Microsoft Docs"
-description: Bing Maps Services API Reference for the Android and iOS SDK
-ms.author: "pablocan"
+title: MapServices API Reference | Microsoft Docs
+description: Provides a reference of MapServices API through tables that outline details for various classes, enumerations, and interfaces.
```
Comment (stevemunk) [inline BingMaps/sdk-native/map-services-api/index.md:3]:
```suggestion
description: The overview page for the MapServices API Reference section contains tables containing descriptions and links to more information for each of the MapServices classes, Enumerations and interfaces.
```

File: BingMaps/sdk-native/map-services-api/maplocationaddress-class.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
-title: "MapLocationAddress Class | Microsoft Docs"
-ms.author: "pablocan"
+title: MapLocationAddress Class | Microsoft Docs
+description: Describes the MapLocationAddress class for Android and iOS and provides the class's properties and additional references.
```
Comment (stevemunk) [inline BingMaps/sdk-native/map-services-api/maplocationaddress-class.md:3]:
```suggestion
description: Describes the MapLocationAddress class for Android and iOS with descriptions for each of its properties.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by dereklh7** [comment]
@stevemunk I've implemented the requested fixes. Please let me know if there are any remaining issues that are preventing approval and merge for this PR. Thanks for looking at all these PRs, Steve! Your work is much appreciated!

**Pull request merged**